index.bs

```diff
@@ -931,46 +931,67 @@ that can be written to, to transmit data to the remote host.
+  readonly attribute Promise&lt;StreamAbortInfo&gt; remoteReset;
+  undefined reset(optional StreamAbortInfo abortInfo = {});
 };
 </pre>
 
-## Overview ##  {#outgoing-stream-overview}
+## Overview ##  {#send-stream-overview}
 
 The {{SendStream}} will be initialized by running the {{WritableStream}}
 initialization steps.
 
-## Attributes ##  {#outgoing-stream-attributes}
+## Attributes ##  {#send-stream-attributes}
 
-: <dfn attribute for="SendStream">writingAborted</dfn>
-:: The `writingAborted` attribute represents a promise that is [=fulfilled=]
-    when the a message from the remote side aborting the stream is received.
-    For QUIC, that message is a STOP_SENDING frame. When the stream receives
-    this mesage, the user agent MUST run the following:
-      1. Let |stream| be the {{SendStream}} object.
-      1. Let |transport| be the {{WebTransport}}, which the |stream| was created
-         from.
-      1. Remove the stream from the |transport|'s [=[[OutgoingStreams]]=].
-      1. [=Resolve=] the promise with the resulting {{StreamAbortInfo}} with
-         the {{StreamAbortInfo/errorCode}} set to the value from the aborting
-         message from the remote side.
+: <dfn attribute for="SendStream">remoteReset</dfn>
+:: The `remoteReset` attribute represents a promise that is [=fulfilled=]
+    when a message from the remote side aborting the stream is received.
+    For QUIC, that message is a STOP_SENDING frame.
 
-## Methods ##  {#outgoing-stream-methods}
+## Methods ##  {#send-stream-methods}
 
-: <dfn method for="SendStream">abortWriting()</dfn>
+: <dfn method for="SendStream">reset()</dfn>
 :: A hard shutdown of the {{SendStream}}. It may be called regardless of
    whether the {{SendStream}} was created by the local or remote peer. When
-   the `abortWriting` method is called, the user agent MUST run the following
+   the `reset` method is called, the user agent MUST run the following
    steps:
      1. Let |stream| be the {{SendStream}} object which is about to abort
         writing.
      1. Let |transport| be the {{WebTransport}}, which the |stream| was created
         from.
-     1. Remove the stream from the transport's [=[[OutgoingStreams]]=].
+     1. Remove |stream| from the transport's [=[[OutgoingStreams]]=].
      1. Let |abortInfo| be the first argument.
-     1. Start the closing procedure by sending a RST_STREAM frame with its
-        error code set to the value of |abortInfo.errorCode|.
+     1. Let |reason| be |abortInfo|.errorCode.
+     1. If it hasn't started already, start the closing procedure by sending a
+        RST_STREAM frame with its error code set to the value of |reason|.
+     1. Let |p| be the result of [=WritableStream/abort|aborting=] |stream|, with |reason|.
+     1. Set |p|.[=[[PromiseIsHandled]]=] to true.
+
+## Procedures ##  {#send-stream-procedures}
+
+When a {{SendStream}} receives a STOP_SENDING frame from the remote side,
```

Comment:
Shouldn't this be RST_STREAM?

Reply:
According to [writingAborted](https://w3c.github.io/webtransport/#dom-sendstream-writingaborted) it's STOP_SENDING. Is that not correct?